framework/src/context_vulkan.cpp

```diff
@@ -674,27 +674,27 @@ namespace gvk
 		if ((flags & VK_DEBUG_REPORT_ERROR_BIT_EXT) != 0) {
 			LOG_ERROR__(fmt::format("Debug Report callback with flags[{}], object-type[{}], and Message[{}]",
 				to_string(vk::DebugReportFlagsEXT{ flags }),
-				to_string(vk::DebugReportObjectTypeEXT{objectType}),
+				to_string(vk::DebugReportObjectTypeEXT(objectType)),
```

Comment:
Oh.. hmm... is this something compiler-specific that only worked on MSVC?

Reply:
Yes, but I'm not sure why. I'd also have written it that way.